src/serverlessPlugin.d.ts

```diff
@@ -46,6 +46,7 @@ export interface AutoSwaggerCustomConfig {
     swaggerPath?: string
     basePath?: string
     schemes?: SwaggerScheme[]
+    excludeStages?: string[]

```

Comment:
Could you add documentation for this in readme.md as well?

Reply:
Those docs look sufficient, thank you.
And thanks @mrcarromesa for review 😄